### PR TITLE
html5-css-form-task

### PR DESCRIPTION
**Pull request by PAtrofimov**
### html5-css-form-task

[Demo](https://patrofimov.github.io/html5-css-form-task/) |
[Code base](https://github.com/PAtrofimov/html5-css-form-task)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Comment by PAtrofimov** [comment]
Fixed all

File: submissions/patrofimov/html5-form/css/styles.css [hunk 1]
```diff
@@ -1055,8 +1052,21 @@ summary {
     color: #76a7e1;
     position: absolute;
     z-index: 1;
-    
-}
+ }
+ 
+ .tone_input:checked + label:after {
```
Comment (A-Ostrovnyy) [inline submissions/patrofimov/html5-form/css/styles.css]: Fix your code formating.

**Comment by PAtrofimov** [comment]
Fixed hover of inputs and formatting css

File: submissions/patrofimov/html5-form/index.html [hunk 2]
```diff
@@ -0,0 +1,651 @@
...
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="main-wrapper">
+        <div class="order">
+          <section class="order_wrapper">
+            <table class="order_table">
+              <caption class="order_caption">
```
Comment (A-Ostrovnyy) [inline submissions/patrofimov/html5-form/index.html]: Check icon size.

File: submissions/patrofimov/html5-form/index.html [hunk 3]
```diff
@@ -0,0 +1,651 @@
...
+              Place order
+            </button>
+            <button type="button" class="order-actions_btn">
+              Print a quote
+            </button>
+          </div>
+
+          <section class="help">
+            <h2 class="help_caption">
+              <span class="help-ring" aria-hidden="true">&#8853;</span>Help
```
Comment (A-Ostrovnyy) [inline submissions/patrofimov/html5-form/index.html]: This text should be uppercase.

File: submissions/patrofimov/html5-form/index.html [hunk 4]
```diff
@@ -0,0 +1,651 @@
...
+          id="translate-part"
+          method="POST"
+          action="https://webhook.site/9f86ffda-62e0-498b-97aa-fe5d72f5888b"
+          class="basic_form"
+        >
+          <section class="translate basic_section">
+            <h1 class="translate_caption">
+              Select what you want to translate
+            </h1>
+            <fieldset class="translate_fieldset">
```
Comment (A-Ostrovnyy) [inline submissions/patrofimov/html5-form/index.html]: This block must be 100% width inside the parent container.
Also, one of the radio buttons must be checked in all sections with radio buttons.

File: submissions/patrofimov/html5-form/index.html [hunk 5]
```diff
@@ -0,0 +1,651 @@
...
+                      class="upload-document upload-table_th_hidden"
+                      aria-hidden="true"
+                      >&#9016;</span
+                    >
+                    business_financial.docx
+                  </td>
+                  <td class="upload-table_td upload-table_th_hidden">26kb</td>
+                  <td class="upload-table_td upload-table_th_hidden">842</td>
+                  <td class="upload-table_td upload-table_td-btn">
+                    <button class="upload-table_btn" type="button">
```
Comment (A-Ostrovnyy) [inline submissions/patrofimov/html5-form/index.html]: Remove the border and background on the trash button.
Change trash icon to more obvious.

File: submissions/patrofimov/html5-form/index.html [hunk 6]
```diff
@@ -0,0 +1,651 @@
...
+            </div>
+          </section>
+
+          <section class="lang basic_section">
+            <h2>Choose your languages pairings</h2>
+            <fieldset class="lang_fieldset">
+              <div class="lang-wrapper">
+                <label class="lang_label" for="lang_select_from">From</label>
+                <div class="lang_select-wrapper">
+                  <select
```
Comment (A-Ostrovnyy) [inline submissions/patrofimov/html5-form/index.html]: Make as close as possible to the mockup. Example of styled select you can take in [my presentation ](https://slides.com/alexander-5/deck/#/)

File: submissions/patrofimov/html5-form/index.html [hunk 7]
```diff
@@ -0,0 +1,651 @@
...
+                    <option value="spanisch">Spanisch</option>
+                  </select>
+                </div>
+              </div>
+
+              <div class="lang-wrapper lang-wrapper_indent">
+                <label class="lang_label" for="lang_select_to">To</label>
+
+                <div class="lang_select-wrapper">
+                  <select
```
Comment (A-Ostrovnyy) [inline submissions/patrofimov/html5-form/index.html]: This `select` is required too.

File: submissions/patrofimov/html5-form/index.html [hunk 8]
```diff
@@ -0,0 +1,651 @@
...
+                    <option value="english">English</option>
+                    <option value="arabic">Arabic</option>
+                    <option selected value="kurdish">Kurdish</option>
+                    <option value="russian">Russian</option>
+                    <option value="spanisch">Spanisch</option>
+                  </select>
+                </div>
+
+                <div class="lang_actions">
+                  <button class="lang_btn" type="button">
```
Comment (A-Ostrovnyy) [inline submissions/patrofimov/html5-form/index.html]: Use the same trash icon as inside the table with the list of file.

File: submissions/patrofimov/html5-form/index.html [hunk 9]
```diff
@@ -0,0 +1,651 @@
...
+                    <span class="lang_trash" aria-hidden="true">&#10062;</span>
+                  </button>
+                  <button class="lang_btn" type="button">
+                    <span class="lang_add" aria-hidden="true">&#10133;</span>
+                    ADD
```
Comment (A-Ostrovnyy) [inline submissions/patrofimov/html5-form/index.html]: Don't use text in uppercase as content. Use CSS for this. When you change text your visual representation should be the same and independent of text content or text case inside HTML.

File: submissions/patrofimov/html5-form/index.html [hunk 10]
```diff
@@ -0,0 +1,651 @@
...
+            </fieldset>
+          </section>
+
+          <div class="basic_section basic_section-wrapper">
+            <section class="tone">
+              <h2 class="tone_caption">Select your desired tone</h2>
+              <fieldset>
+                <div class="tone-actions">
+                  <div class="tone-actions_btn">
+                    <input
```
Comment (A-Ostrovnyy) [inline submissions/patrofimov/html5-form/index.html]: Add icon before text as on the mockup.

File: submissions/patrofimov/html5-form/index.html [hunk 11]
```diff
@@ -0,0 +1,651 @@
...
+                  </div>
+                </div>
+              </fieldset>
+            </section>
+
+            <section class="urgent">
+              <h2 class="urgent_caption">Tell us how urgent this job is</h2>
+              <fieldset>
+                <div class="urgent-timeline">
+                  <input
```
Comment (A-Ostrovnyy) [inline submissions/patrofimov/html5-form/index.html]: Dots on the edges must have the same height as `-webkit-slider-runnable-track`.

File: submissions/patrofimov/html5-form/index.html [hunk 12]
```diff
@@ -0,0 +1,651 @@
...
+                    >average</label
+                  >
+                  <label
+                    class="urgent-timeline_label"
+                    for="urgent-timeline_range"
+                    >yesterday</label
+                  >
+                </div>
+                <details open class="timeline-info">
+                  <summary class="timeline-info_caption">timeline info</summary>
```
Comment (A-Ostrovnyy) [inline submissions/patrofimov/html5-form/index.html]: This is just heading. In this case `<summary>` tag is unnecessary.

File: submissions/patrofimov/html5-form/index.html [hunk 13]
```diff
@@ -0,0 +1,651 @@
...
+                  id="payment-amazon"
+                  class="required-input visually-hidden"
+                />
+                <label class="payment_radio_label" for="payment-amazon">
+                  amazon
+                  <img src="img/amazon.jpg" alt="amazon" />
+                </label>
+              </div>
+
+              <div class="payment_card">
```
Comment (A-Ostrovnyy) [inline submissions/patrofimov/html5-form/index.html]: Check sizes and proportions of cards info inputs.

File: submissions/patrofimov/html5-form/index.html [hunk 14]
```diff
@@ -0,0 +1,651 @@
...
+              <div class="payment-actions">
+                <div class="payment-agree payment-agree_indent">
+                  <input
+                    required
+                    type="checkbox"
+                    class="payment-agree_input"
+                    id="payment-agree_input"
+                  />
+                  <label for="payment-agree_input"
+                    >I agree to the <a href="#">Terms & Conditions</a></label
```
Comment (A-Ostrovnyy) [inline submissions/patrofimov/html5-form/index.html]: Remove link from a label. When you click on the link your checkbox will be checked. Do we need this behavior?)

File: submissions/patrofimov/html5-form/index.html [hunk 15]
```diff
@@ -0,0 +1,651 @@
...
+            </fieldset>
+          </section>
+        </form>
+      </div>
+    </main>
+
+    <footer class="document_footer">
+      <div class="footer-wrapper">
+        <section class="what-we-do">
+          <h4 class="footer_caption">
```
Comment (A-Ostrovnyy) [inline submissions/patrofimov/html5-form/index.html]: Align the headings and text with a vertical line.

File: submissions/patrofimov/html5-form/index.html [hunk 16]
```diff
@@ -0,0 +1,651 @@
...
+            <span
+              class="fa fa-compass fa-lg fa-footer"
+              aria-hidden="true"
+            ></span>
+            Our main office
+          </h4>
+          <address class="footer_text">
+            Dubai, UAE <br />
+            P.O. Box: 502735 <br />
+            Phone: +971(0) 44278128
```
Comment (A-Ostrovnyy) [inline submissions/patrofimov/html5-form/index.html]: Use link as on the form header ```<a href="tel:+971(0)555555">+971(0)555 555</a>```

File: submissions/patrofimov/html5-form/index.html [hunk 17]
```diff
@@ -0,0 +1,651 @@
...
+            <span
+              class="fa fa-share-alt fa-lg fa-footer"
+              aria-hidden="true"
+            ></span>
+            Connect with us
+          </h4>
+          <ul class="footer_list">
+            <li>
+              <a href="#"
+                ><span
```
Comment (A-Ostrovnyy) [inline submissions/patrofimov/html5-form/index.html]: Add icons before the text.

**Comment by PAtrofimov** [comment]
Fixed all.

**Comment by A-Ostrovnyy** [comment]
Good work!
Check email validation. [Image](https://imgur.com/a/mBN2pV6)

**Comment by PAtrofimov** [comment]
Fixed everything

**Comment by A-Ostrovnyy** [comment]
Well done!

**Pull request merged**